cl/users/admin.py

```diff
@@ -19,30 +21,42 @@
     UserProfile,
 )
 
+UserProxyEvent = apps.get_model("users", "UserProxyEvent")
+UserProfileEvent = apps.get_model("users", "UserProfileEvent")
 
-def get_email_confirmed(obj):
-    return obj.profile.email_confirmed
-
-
-get_email_confirmed.short_description = "Email Confirmed?"
 
+class TokenInline(admin.StackedInline):
+    model = Token
 
-def get_stub_account(obj):
-    return obj.profile.stub_account
 
+class UserProfileInline(admin.StackedInline):
+    model = UserProfile
 
-get_stub_account.short_description = "Stub Account?"
 
+class CustomUserChangeForm(UserChangeForm):
+    class Meta(UserChangeForm.Meta):
+        model = User
+        fields = "__all__"
```

Comment:
Could we omit overriding the class Meta here? I think it doesn't change from the parent Meta.